firefox_puppeteer/firefox_puppeteer/api/software_update.py

```diff
@@ -7,9 +7,9 @@
 
 import mozinfo
 
+from firefox_puppeteer.base import BaseLib
 from firefox_puppeteer.api.appinfo import AppInfo
 from firefox_puppeteer.api.prefs import Preferences
-from firefox_puppeteer.base import BaseLib
 
```

Comment:
Basically this is true, but given that this is a plain backport, we should get this fixed on the mozilla-central branch first.